components/admin/requests/Footer.tsx

```diff
@@ -0,0 +1,89 @@
+import { Box, HStack, Text, Button, Link, Stack } from '@chakra-ui/react'; // Chakra UI
+import ApproveRequestModal from '@components/admin/requests/processing/ApproveRequestModal'; // Approve button + modal
+import RejectRequestModal from '@components/admin/requests/processing/RejectRequestModal'; // Reject button + modal
+import { ApplicationStatus } from '@lib/graphql/types';
+import Footer from '@components/admin/Footer';
+import CompleteRequestModalButton from './processing/CompleteModalButton';
+
+type RequestFooterProps = {
+  readonly applicationId: number;
+  readonly applicationStatus?: ApplicationStatus;
+  readonly allStepsCompleted: boolean;
+  readonly applicantId?: number;
+};
+
+/**
+ * Footer of View Request page
+ * @param applicationId ID of application
+ * @param applicationStatus Status of application creation
+ * @param allStepsCompleted Whether all processing tasks are complete
+ * @param applicantId ID of applicant
+ */
+export default function RequestFooter({
```

Comment:
I consolidated the Footer logic that was in `components/admin/requests/Header.tsx` and `components/admin/requests/processing/ProcessingFooter.tsx` into a Footer component since it will always be visible now.